cmd/fleetctl/testdata/gitops/lib/query_multiple.yml

```diff
@@ -1,11 +1,4 @@
-apiVersion: v1
-kind: query
-spec:
-  name: query_ruby
+- name: query_ruby
   query: select 1
----
-apiVersion: v1
-kind: query
-spec:
-  name: query_ruby2
+- name: query_ruby2
   query: select 2
```

Comment:
I assumed because of the way the bug was written that the old format was a mistake and that we don't want to support it, but I should double check that

Reply:
it's just because people might already be using it. We are using it (but I don't think it's a big deal to break it for us)

if we're okay breaking, could you also (separate PR is fine) update the yaml in the `it-and-security` folder of the repo? 🙏

Reply:
I re-added support for the old `apply` format, better to not potentially break things for customers 🙂